subsystems/win/basesrv/vdm.c

```diff
@@ -904,7 +904,7 @@ CSR_API(BaseSrvUpdateVDMEntry)
             case VdmEntryUpdateControlCHandler:
             {
                 // TODO: NOT IMPLEMENTED
-                DPRINT1("BaseSrvUpdateVDMEntry: VdmEntryUpdateControlCHandler not implemented!");
+                DPRINT1("BaseSrvUpdateVDMEntry: VdmEntryUpdateControlCHandler not implemented\n");
```

Comment:
```suggestion
                DPRINT1("BaseSrvUpdateVDMEntry: VdmEntryUpdateControlCHandler unimplemented\n");
```

Reply:
I would agree, even to do more, but there are multiple similar cases, even in this very file,
so that will wait for a separate PR, or more likely just stay as is for the time being.

Reply:
I am ok with doing that in a separate PR.